src/views/Home.vue

```diff
@@ -3,16 +3,16 @@
 		class="bg-primary"
 		data-e2e-id="homePage"
 	>
-		<div class="flex bg-gray-100 py-48 skewed-bottom-wrapper">
-			<div class="flex flex-col w-1/2 justify-center items-center">
-				<h1 class="font-bold text-6xl">
+		<div class="block lg:flex bg-gray-100 py-48 skewed-bottom-wrapper">
+			<div class="flex flex-col lg:w-1/2 justify-center items-center">
+				<h1 class="font-bold text-6xl ">
```

Comment:
The heading still looks too big.

![image](https://user-images.githubusercontent.com/27869979/98372739-87014280-203e-11eb-97e3-53deeffd5853.png)
![image](https://user-images.githubusercontent.com/27869979/98372744-88326f80-203e-11eb-988b-454ddb402783.png)